src/common/common.h

```diff
@@ -14,9 +15,10 @@ std::optional<POINT> get_mouse_pos();
 // Gets active window, filtering out all "non standard" windows like the taskbar, etc.
 HWND get_filtered_active_window();
 // Gets window ancestor (usualy the window we want to do stuff with), filtering out all "non standard" windows like the taskbar, etc. and provide the app process path
-struct WindowAndProcPath {
-  HWND hwnd = nullptr;
-  std::wstring process_path;
+struct WindowAndProcPath
```

Comment:
Can you remove those unrelated style changes? We should do the styling in a separate PR.